deluca/envs/lung/_balloon_lung.py

```diff
@@ -73,6 +73,7 @@ def __init__(
         self.reset()
 
     def reset(self):
+        self.time = 0.0
```

Comment:
Is this necessary? Thought it was perhaps in the base `lung` environment? Could be mistaken

Reply:
Well originally I added this because for loop and scan weren't agreeing even after lung.reset(), so I think it is necessary. Also, the base lung just has a def render method and nothing else.